doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc

```diff
@@ -40,31 +40,34 @@ If you want to enable LDAP failover, you can set two or more LDAP server address
 --
 . If you selected `RH-SSO` or `LDAP`, if your RH-SSO or LDAP system does not define all the roles required for your deployment, you can map authentication system roles to {PRODUCT} roles.
 +
-To enable role mapping, you must provide a role mapping configuration file in an OpenShift configuration map or secret object in the project namespace. The file must contain entries in the following format:
+To enable role mapping, you must provide role mapping as a single configuration string, or else a role mapping configuration file in an OpenShift configuration map or secret object in the project namespace.
++
+The string must use the `role=role1,role2;another-role=role2` pattern, for example: `admins=kie-server,rest-all,admin;developers=kie-server,rest-all`.
++
+The file must contain entries in the following format:
 +
 [subs="attributes,verbatim,macros"]
 ----
-ldap_role = product_role1, product_role2...
+ldap_role=product_role1, product_role2...
 ----
 +
 For example:
 +
 [subs="attributes,verbatim,macros"]
 ----
-admins = kie-server,rest-all,admin
+admins=kie-server,rest-all,admin
 ----
 +
-To enable the use of this file, make the following changes:
+To enable the use of this string or file, make the following changes:
 +
-.. Under *RoleMapper*, in the *Roles properties file* field, enter the fully qualified path name of the role mapping configuration file, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`.
-.. If you want to replace roles defined in the authentication system with roles that you define in the mapping file, select the *Replace roles* box. Otherwise, both the roles defined in RH-SSO or LDAP and the roles defined in the configuration file are available.
-.. In the fields under *RoleMapper Configuration object*, select the *Kind* of the object that provides the file (`ConfigMap` or `Secret`) and enter the *Name* of the object. This object is automatically mounted on {CENTRAL} and {KIE_SERVER} pods in the path that you specified for the role mapping configuration file.
+.. Under *RoleMapper*, in the *Roles properties file* field, enter the role configuration string or the fully qualified path name of the role mapping configuration file, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`.
+.. If you define role mapping, by default only the roles that you define in the mapping are available. If you want to keep the original roles that are defined in the authentication system and that your mapping maps to other roles, select the *Roles keep mapped* box. If you want to keep the original roles that are defined in the authentication system and not mentioned in your mapping,select the *Roles keep non mapped* box.
```

Comment:
You need to start all steps with an imperative.You can explain after the imperative. So I suggest something like this, but feel free to use your own version:

b. Select the Roles keep mapped box or the  Roles keep non mapped box. If you define role mapping, by default only the roles that you define in the mapping are available. If you want to keep the original roles that are defined in the authentication system and that your mapping maps to other roles, select the Roles keep mapped box. If you want to keep the original roles that are defined in the authentication system and not mentioned in your mapping,select the Roles keep non mapped box.